pkg/testutils/kustainer/kustainer.go

```diff
@@ -260,3 +261,14 @@ func (c *KustainerContainer) CreateDatabase(ctx context.Context, dbName string)
 	}
 	return nil
 }
+
+type threadsafeBuffer struct {
+	sync.Mutex
+	buffer bytes.Buffer
+}
+
+func (b *threadsafeBuffer) Write(p []byte) (n int, err error) {
```

Comment:
Do we need a wrapper to read the buffer? I think we probably need to have something that wraps reads with a lock operation too.

Reply:
I don't think so, but I added one.